sw/device/lib/testing/test_framework/ottf.h

```diff
@@ -44,6 +42,12 @@ typedef struct test_config {
    * not require concurrency, and seek to minimize simulation runtime.
    */
   bool enable_concurrency;
```

Comment:
It may be worth adding some static assertions to check the offset and size of this field to catch undesired layout changes. We've added a `static_assert` macro in `assert.h` that you can use for this.

Reply:
Great idea! Fixed.

Reply:
You may also want to take a look at `OT_ASSERT_SIZE` and `OT_ASSERT_MEMBER_OFFSET` in `sw/device/lib/base/macros.h`.

Reply:
Thanks @alphan! I addressed in this #9510. Marking as fixed here.